org.dpsoftware.FireflyLuciferin.json

```diff
@@ -0,0 +1,106 @@
+{
+  "id": "org.dpsoftware.FireflyLuciferin",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "/app/fireflyluciferin/bin/FireflyLuciferin",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=pulseaudio",
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=all",
```

Comment:
because my app uses GPU and tty devices. it's an ambilight clone app that uses external devices to drive a let strip along the GPU to capture the screen.

Reply:
I still have no idea what "ambilight clone app" so it's hard to guess.

> it supports wayland and x11.

Then what I said.

Reply:
see https://github.com/flathub/flathub/pull/5874#discussion_r1854894690

Reply:
ok removed wayland, it seems that X11 socket is enough.